pom.xml

```diff
@@ -29,6 +29,10 @@
     <tag>HEAD</tag>
   </scm>
 
+  <properties>
+    <dep.kafka.version>0.10.1.0</dep.kafka.version>
+  </properties>
+
```

Comment:
Is the logging cluster the only one that's still on 9?  Otherwise, this won't work—connecting newer clients to older brokers may still be busted.

- [KAFKA-3600][2]
- [KAFKA-3601][1]

[1]: https://issues.apache.org/jira/browse/KAFKA-3601
[2]: https://issues.apache.org/jira/browse/KAFKA-3600